src/states/node.rs

```diff
@@ -940,7 +940,7 @@ impl Node {
             events.add_event(Event::NodeAdded(*name, self.peer_mgr.routing_table().clone()));
         }
         self.is_approved = true;
-
+        let _ = self.send_rt_request();
```

Comment:
as discussed, the add_prefixes will be refactored and handle_node_approval will handle the error and restart the node